tests/attestation_plugin_cert/host/host.cpp

```diff
@@ -0,0 +1,219 @@
+                    security_version);
+                goto done;
+            }
+        }
+        // Dump an enclave's unique ID, signer ID and Product ID. They are
+        // MRENCLAVE, MRSIGNER and ISVPRODID for SGX enclaves. In a real
+        // scenario, custom id checking should be done here
+        else if (
+            strcmp(claim->name, OE_CLAIM_SIGNER_ID) == 0 ||
+            strcmp(claim->name, OE_CLAIM_UNIQUE_ID) == 0 ||
+            strcmp(claim->name, OE_CLAIM_PRODUCT_ID) == 0)
+        {
+            OE_TRACE_INFO("Enclave %s:\n", claim->name);
+            for (size_t j = 0; j < claim->value_size; j++)
+            {
+                OE_TRACE_INFO("0x%0x ", claim->value[j]);
+            }
+        }
+    }
+
+    result = OE_OK;
+done:
+    return result;
+}
+
+void run_test(oe_enclave_t* enclave, int test_type)
+{
+    oe_result_t result = OE_FAILURE;
+    oe_result_t ecall_result;
+    unsigned char* cert = NULL;
+    size_t cert_size = 0;
+
+    OE_TRACE_INFO(
+        "Host: get tls certificate signed with %s key from an enclave \n",
+        test_type == TEST_RSA_KEY ? "a RSA" : "an EC");
+    if (test_type == TEST_EC_KEY)
+    {
+        result = get_tls_cert_signed_with_ec_key(
+            enclave, &ecall_result, &cert, &cert_size);
+    }
+    else if (test_type == TEST_RSA_KEY)
+    {
+        result = get_tls_cert_signed_with_rsa_key(
+            enclave, &ecall_result, &cert, &cert_size);
+    }
+
+    if ((result != OE_OK) || (ecall_result != OE_OK))
+        oe_put_err(
+            "get_tls_cert_signed_with_%s_key() failed: result=%u",
+            test_type == TEST_RSA_KEY ? "rsa" : "ec",
+            result);
+
+    fflush(stdout);
+
+    {
+        // for testing purpose, output the whole cer in DER format
+        char filename[FILENAME_LENGTH];
+        FILE* file = NULL;
+
+        sprintf_s(
+            filename,
+            sizeof(filename),
+            "./cert_%s.der",
+            test_type == TEST_RSA_KEY ? "rsa" : "ec");
+        OE_TRACE_INFO(
+            "Host: Log quote embedded certificate to file: [%s]\n", filename);
+#ifdef _WIN32
+        fopen_s(&file, filename, "wb");
+#else
+        file = fopen(filename, "wb");
+#endif
+        fwrite(cert, 1, cert_size, file);
+        fclose(file);
+    }
+
+    // validate cert
+    OE_TRACE_INFO("Host: Verifying tls certificate\n");
+    OE_TRACE_INFO("Host: cert = %p cert_size = %d\n", cert, cert_size);
+    result = oe_verify_attestation_certificate_with_evidence(
+        cert, cert_size, enclave_claims_verifier, NULL);
+    OE_TRACE_INFO(
+        "Host: Verifying the certificate from a host ... %s\n",
+        result == OE_OK ? "Success" : "Fail");
+    fflush(stdout);
+    OE_TEST(result == OE_OK);
+
+    OE_TRACE_INFO("free cert 0xx%p\n", cert);
+    // oe_verifier_shutdown();
+    free(cert);
+}
+
+int main(int argc, const char* argv[])
+{
+#ifdef OE_LINK_SGX_DCAP_QL
```

Comment:
Other PRs remove this define.  See PR #3273 and #3270 so will need to update this as they are merged

Reply:
Thanks for the info. Removed.